doc/ref_arch/openstack/chapters/chapter07.md

```diff
@@ -27,16 +29,74 @@
+
+Logs have multiple operational uses including for:
+
+1. Regulatory Compliance and SIEM featuring the automated gathering, analysis and correlation of log data across all systems and devices across AT&T to provide real-time analysis, event prioritization, reporting, notification and alerting.
+1. Monitoring across systems in real-time to detect particular log events, patterns, anomalies or inactivity to gauge system and application health
+1. Identify system and application performance and configuration issues
+1. Root cause analysis for system and application failures and errors
+1. Ensuring that operational objectives and SLAs are met
+
+
+### 7.5.2. Monitoring
+Monitoring is the process of collecting, aggregating, and analyzing values that improve awareness of 
+the components' characteristics and behavior. The data from various parts of the environment are collected 
+into a monitoring system that is responsible for storage, aggregation, visualization, and initiating automated 
+responses when the values meet specific threshold.
+
+Monitoring systems fulfill many related functions. Their first responsibility is to accept and store incoming 
+and historical data. While values representing the current point in time are useful, it is almost always more 
+helpful to view those numbers in relation to past values to provide context around changes and trends. 
+
+### 7.5.2. Alerting
+Alerting is the responsive component of a monitoring system that performs actions based on changes in metric 
+values. Alert definitions are composed of two components: a metrics-based condition or threshold, and an 
+action to perform when the values fall outside of the acceptable conditions.
+
+While monitoring systems are incredibly useful for active interpretation and investigation, one of the primary
+benefits of a complete monitoring system is letting administrators disengage from the system. ALerts allow the
+specification of situations that make sense to actively manage, while relying on the passive monitoring of the 
+software to watch for changing conditions.
+
+### 7.5.3. Logging,Monitoring, and Analytics (LMA) Framework
+In this section, a possible framework utilizing Prometheus, Elasticsearch and Kibana is given as an example only.
+
+
+<p align="center"><img src="../figures/Figure_4_4_Monitoring_Logging_Framework.png" alt="Monitoring and Logging Framework"></br>
+Figure 7-1: Monitoring and Logging Framework </p>
+
+The monitoring and logging framework (**Figure 7-1**) leverages Prometheus as the monitoring engine and 
+Fluentd for logging. In addition, the framework uses Elasticsearch to store and organize logs for easy access. 
+Prometheus agents pull information from individual components on every host.  Fluentd, an open source data 
+collector, unifies data collection and consumption for better use and understanding of data. Fluentd captures 
+the access, application and system logs.
```

Comment:
Where do we reference PM - Performance Management?   Do we have a suggested model with collectd and some central collection?

Reply:
@iangardner22 Atthis time we only have requirements but we do need a model as you have suggested.